object_detection/object_detection_tutorial.ipynb

```diff
@@ -100,8 +103,13 @@
    },
    "outputs": [],
    "source": [
+    "# What model to download.\n",
```

Comment:
Can you point to the detection model zoo too?

https://github.com/tensorflow/models/blob/master/object_detection/g3doc/detection_model_zoo.md